src/assets/images/arrow-down.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="6" viewBox="0 0 12 6" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can't we reuse already existing `arrow-*.svg` images?

Reply:
@asvishnyakov already existed ones have different color, so there are two ways:
1) We are using new ones and removing the old ones from assets
2) We can use old ones and then write custom css to change the color

Reply:
done, removed unused arrow icons